external-crates/move/crates/move-compiler/src/diagnostics/mod.rs

```diff
@@ -440,6 +453,30 @@ macro_rules! diag {
     }};
 }
 
+#[macro_export]
+macro_rules! ice {
+    ($primary: expr $(,)?) => {{
+        $crate::diagnostics::print_stack_trace();
+        diag!($crate::diagnostics::codes::Bug::ICE, $primary)
+    }};
+    ($primary: expr, $($secondary: expr),+ $(,)?) => {{
+        $crate::diagnostics::print_stack_trace();
```

Comment:
Would it make sense to add the stack trace as a diagnostic note?

Reply:
Maybe? No strong feelings here. Though it might help with folks that send us bug reports? Unsure

Reply:
Yeah, I was thinking it would be useful to include with bug reports, hence why I want it on the diagnostic form.